### PR TITLE
chore: update getting started docs

### PR DESCRIPTION
**Pull request by mlogsdon18**
## Description

[CSS-561](https://jira.corp.adobe.com/browse/CSS-561)

This PR updates the Getting Started page on the docs site to include more detailed and updated information that would allow someone to get a Button component added to their project.

## How and where has this been tested?

Please tag yourself on the tests you've marked complete to confirm the tests have been run by someone other than the author.

- [ ] Walk through the steps on the [Getting Started page](https://pr-2081--spectrum-css.netlify.app/get-started) and confirm that they are complete and thorough

### Regression testing

Validate:

1. A legacy documentation page (such as [accordion](https://pr-2081--spectrum-css.netlify.app/accordion.html)), including:

- [ ] The page renders correctly
- [ ] The page is accessible
- [ ] The page is responsive

2. A migrated documentation page (such as [action group](https://pr-2081--spectrum-css.netlify.app/actiongroup.html)), including:

- [ ] The page renders correctly
- [ ] The page is accessible
- [ ] The page is responsive

## To-do list

- [x] I have read the [contribution guidelines](/.github/CONTRIBUTING.md).

- [x] If my change impacts **documentation**, I have updated the documentation accordingly.

- [ ] ✨ This pull request is ready to merge. ✨

### CODE REVIEW

File: site/get-started.pug
Comment (GarthDB) [inline site/get-started.pug:75]: Doesn't the button need `@spectrum-css/tokens` package and not the `@spectrum-css/vars` package? It's included in the `npm install` above, but I don't see it used anywhere.

**Review by GarthDB — CHANGES_REQUESTED** [review]
When the migration is done, will we still need to include the `@spectrum-css/vars` package? Do we need to include instructions on how to know when to use `@spectrum-css/vars` and when to use `@spectrum-css/tokens`?

File: site/get-started.pug
Comment (mlogsdon18) [reply site/get-started.pug:75]: Yep you're right that was just something I forgot to remove. @pfulton I know we're still in the process of migrating a few components but are you ok with removing the vars package from the instructions and only including tokens?

Comment (pfulton) [reply site/get-started.pug:75]: Yeah, it might be time to go Spectrum Tokens-only—we're very close to being completely switched over. @castastrophe what do you think?

We could also hold this for a little bit until the migration work is complete.

**Comment by pfulton** [comment]
Holding this one for after the build refactor work is complete.

**Comment by castastrophe** [comment]
Reminder to me @castastrophe to resolve this after we merge the build work.

**Comment by castastrophe** [comment]
Incorporated these updates in the updated 11ty work in #2093

**Pull request closed without merging**
